modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,30 +5,112 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== Introduction to SSO
+
+Single Sign-On (SSO) allows for both keeping data safe and providing authorized users safe access to the data.
+This section describes how to set up Aura SSO with Role-Based Access Control (RBAC) for centrally controlled privileges as well as data access.
+
+Aura Enterprise supports SSO at the organization, project, and instance levels.
+Aura admins can configure organization and project level SSO while Aura Support can configure instance level SSO.
+
+== Requirements
 
-[NOTE]
-====
 Accessing Aura with SSO requires:
  
-* Authorization Code Flow with PKCE.
+* Authorization Code Flow with PKCE (Proof Key for Code Exchange).
 * A publicly accessible Identity Provider (IdP) server.
-====
+
+== Configuring SSO 
+
+Users can configure a combination of one or more of the following login methods:
+
+* Email/password
+* Okta
+* Microsoft Entra ID (previously Azure Active Directory)
+
+An organization's admin can add Aura as a login from a tile in an organization's Apps Dashboard.
+
+
+== SSO setup requirements 
+
+To configure SSO from Aura, go to the *Aura Console > Settings > SSO Configuration.*
+
+The set-up form requires the following info:
+
+* SSO config ID - you decide a name for this
+* IdP type (e.g. Okta / Microsoft Entra ID)
+* Channel
+* Client ID 
+* Client Secret
+* Discovery URI
+* Issuer
+* Authorization Endpoint
```

Comment:
You need either:
- discovery uri

or 
- Issuer	
- Authorization Endpoint	
- Token Endpoint	
- JWKS URI

all of Issuer Authorization Endpoint Token Endpoint JWKS URI are provided in the discovery uri, so they need either or.